### PR TITLE
Bump glbc version to 0.9.2

### PR DESCRIPTION
**Pull request by timstclair**
Follow up to https://github.com/kubernetes/ingress/pull/384

### CODE REVIEW

File: cluster/saltbase/salt/l7-gcp/glbc.manifest [hunk 1]
```diff
@@ -12,7 +12,7 @@ spec:
   terminationGracePeriodSeconds: 600
   hostNetwork: true
   containers:
-  - image: gcr.io/google_containers/glbc:0.9.1
```
Comment (csbell) [inline cluster/saltbase/salt/l7-gcp/glbc.manifest]: There's two other references to 0.9.1 above -- name and labels.

**Review by csbell — COMMENTED** [review]
Two nits.

File: cluster/saltbase/salt/l7-gcp/glbc.manifest (hunk 1 above)
Comment (timstclair) [inline cluster/saltbase/salt/l7-gcp/glbc.manifest]: Done. Thanks.

**Comment by timstclair** [comment]
@zmerlynn for approval

**Review by nicksardo — APPROVED** [review]

**Comment by timstclair** [comment]
@k8s-bot kubemark e2e test this
@k8s-bot non-cri e2e test this, issue https://github.com/kubernetes/kubernetes/issues/42646
@k8s-bot cvm gce e2e test this, issue https://github.com/kubernetes/kubernetes/issues/42639

**Comment by nicksardo** [comment]
@k8s-bot cvm gce e2e test this, issue #42639

**Pull request merged**